src/Symfony/Component/Form/FormTypeGuesserChain.php

```diff
@@ -25,11 +25,15 @@ class FormTypeGuesserChain implements FormTypeGuesserInterface
      *
      * @throws UnexpectedTypeException if any guesser does not implement FormTypeGuesserInterface
      */
-    public function __construct(array $guessers)
+    public function __construct($guessers)
     {
+        if (!is_array($guessers) && !$guessers instanceof \Traversable) {
+            throw new UnexpectedTypeException($guessers, 'array or Traversable');
+        }
+
         foreach ($guessers as $guesser) {
             if (!$guesser instanceof FormTypeGuesserInterface) {
-                throw new UnexpectedTypeException($guesser, 'Symfony\Component\Form\FormTypeGuesserInterface');
+                throw new UnexpectedTypeException($guesser, FormTypeGuesserInterface::class);
```

Comment:
Problem is that's causing merge conflicts when merging older branches in the newer onces.